src/libmongoc/doc/installing.rst

```diff
@@ -32,19 +32,23 @@ The libmongoc package is available on recent versions of Debian and Ubuntu.
 
 .. code-block:: none
 
-  $ apt-get install libmongoc-1.0-0
+  $ apt-get install libmongoc-dev
+
+The ``libmongoc-dev`` package includes headers required to build applications using libmongoc. To check which version is available, see `https://packages.debian.org/stable/libmongoc-dev <https://packages.debian.org/stable/libmongoc-dev>`_. To only install the libraries without headers, install the ``libmongoc-1.0-0`` package.
 
 On Fedora, a mongo-c-driver package is available in the default repositories and can be installed with:
 
 .. code-block:: none
 
-  $ dnf install mongo-c-driver
+  $ dnf install mongo-c-driver-devel
+
+The ``mongo-c-driver-devel`` package includes headers required to build applications using libmongoc. To check which version is available, see `https://packages.fedoraproject.org/pkgs/mongo-c-driver/mongo-c-driver-devel/ <https://packages.fedoraproject.org/pkgs/mongo-c-driver/mongo-c-driver-devel/>`_. To only install the libraries without headers, install the ``mongo-c-driver`` package.
```

Comment:
rST recognizes inline URIs and transforms them into hyperlinks automatically, so you can omit the explicit link markup.

```suggestion
The ``mongo-c-driver-devel`` package includes headers required to build applications using libmongoc. To check which version is available, see https://packages.fedoraproject.org/pkgs/mongo-c-driver/mongo-c-driver-devel/. To only install the libraries without headers, install the ``mongo-c-driver`` package.
```

Reply:
Thank you for the tip! Suggestion was applied on this link and others.